pkg/views/registry/list/view.go

```diff
@@ -37,26 +37,6 @@ func ListRegistry(registry []*models.Registry) {
 		})
 	}
 
-	// t := table.New(
```

Comment:
Please donot remove this code. We may require it sometime

Reply:
keeping commented-out code in the codebase can lead to confusion and clutter over time.

Since we use Git for version control, any code that we remove can always be retrieved from the repository history if needed. This allows us to maintain a clean and readable codebase while still having access to previous versions of the code if we ever need to reference it.

Reply:
If there's a specific reason why you think we might need this code in the future, let's discuss it

Reply:
Please delete it; we don’t need it. My apologies—I forgot that I also created a utility for the table list.
https://github.com/goharbor/harbor-cli/blob/main/pkg/views/base/tablelist/model.go#L14